modules/acquia_cms_common/modules/acquia_cms_support/acquia_cms_support.info.yml

```diff
@@ -2,6 +2,6 @@ name: "Acquia CMS Support"
 package: "Acquia CMS"
 description: "Compare the config that ships with Acquia CMS with the active config on the site."
 type: module
-core_version_requirement: ^9.5 || ^10
```

Comment:
@rajeshreeputra why are we dropping support of drupal 9?

Reply:
This branch does not support D9.